TODO

```diff
@@ -0,0 +1,19 @@
+CJK:
+* 59-family-prefer-lang:
+  + Google Noto embeds Adobe Source Code Pro as Latin.
```

Comment:
I think the change in this commit does not work as written here.

Moreover, this change may broke font compatibility. Suppose that we have a document using Noto Sans JP and edited on OS like Ubuntu, macOS or Windows. When we open the document on openSUSE, since the Latin glyphs are replaced with Noto Sans, which have different metrics with the original, the layout of document may broke.